core/config/engine.cpp

```diff
@@ -137,11 +137,20 @@ Dictionary Engine::get_version_info() const {
 	return dict;
 }
 
-static Array array_from_info(const char *const *info_list) {
+static Array array_from_info(const char *const *info_list, int anonymous_count = 0) {
 	Array arr;
 	for (int i = 0; info_list[i] != nullptr; i++) {
 		arr.push_back(String::utf8(info_list[i]));
 	}
+	if (anonymous_count > 0) {
+		// This function is technically not only used for donors.
+		// But currently only donors have anonymous entries.
+		if (anonymous_count == 1) {
+			arr.push_back("And 1 anonymous donor");
+		} else {
+			arr.push_back(vformat("And %d anonymous donors", anonymous_count));
+		}
```

Comment:
This looks easy to use a ternary operator for, but that's just a style choice for whoever isn't afraid of horizontal space.
```suggestion
		arr.push_back(anonymous_count == 1 ? "And 1 anonymous donor" : vformat("And %d anonymous donors", anonymous_count));
```